src/types/OpsLevelData.ts

```diff
@@ -15,12 +15,51 @@ export interface OpsLevelServiceData {
           level: { name: string },
           category: { name: string }
         }>,
+      },
+      serviceStats: {
+        rubric: {
+          checkResults: {
+            byLevel: {
+              nodes: Array<LevelCheckResult>
+            }
+          }
+        }
+      },
+      checkStats: {
+        totalChecks: number,
+        totalPassingChecks: number
       }
-    },
+    }
+  }
+}
 
+export type LevelCheckResult = {
```

Comment:
sorry, I was on the wrong line with my original response... I see it as a gray area because it's being used as an array above, which would make that array a CheckResultss then I guess 😂  but yeah I changed it